intel_extension_for_transformers/llm/quantization/nn/modules.py

```diff
@@ -144,16 +138,19 @@ def forward(self, x: torch.Tensor):
 
     def set_weights_bias(self, weight_data, bias=None):
         shape = weight_data.shape
-        weight = torch.ops.bestlaop.woq_quantize(
-            weight_data,
-            True,
-            self.blocksize,
-            self.compute_dtype if self.compute_dtype is not None else "fp32",
-            self.weight_dtype,
-            self.scale_dtype if self.scale_dtype is not None else "fp32",
-            False,
-        )
-        weight.resize_(shape)
+        if weight_data.is_meta:
+            weight = torch.ones(shape, dtype=torch.int8)
+        else:
+            weight = torch.ops.bestlaop.woq_quantize(
+                weight_data,
+                True,
+                self.blocksize,
+                self.compute_dtype if self.compute_dtype is not None else "fp32",
+                self.weight_dtype,
+                self.scale_dtype if self.scale_dtype is not None else "fp32",
+                False,
+            )
+            weight.resize_(shape)
```

Comment:
direct call `resize_` here will waste memory, it's an known issue, i think we may have a better solution.
qbits-backend can provide a function which can acquire the bytes size of the `packweight`, frontend can call it and clip the `packweight` tensor after some shape-safety-checking.
so the whole pipline would be: 
**store**: qbits-quantize -> resize_ for shape-safety-checking -> warm_up inference(we clip `packweight` tensor here) -> inference -> store
**load**: load `packweight` -> resize_ -> warm_up inference(we clip `packweight` tensor here) -> inference
is this solution resonable? cc @PenghuiCheng

Reply:
I think we should create a qweight to save quantized weight.

Reply:
improved the weight resize_